tests/unittest/pywbem/test_indicationlistener.py

```diff
@@ -988,7 +988,7 @@ def test_WBEMListener_incorrect_headers(desc, headers, exp_status, exp_headers):
         </CIM>""",
         400,
         {
-            'CIMErrorDetails': r'Element .CIM. missing required '
+            'CIMErrorDetails': r'Element .CIM. does not have required '
```

Comment:
Actually I would prefer the original text with the word missing to "does not have". There are several cases of this

Reply:
Missing is an emotion. Does the element really miss its kids?

Reply:
We agreed to say "is missing"

Reply:
DONE